packages/auth/src/auth/auth.ts

```diff
@@ -630,12 +642,16 @@ const persistToken = async (jsonResponse: TokenJSONResponse) => {
 
   const { clientId, clientUniqueKey, scopes } = state.credentials;
 
+  const grantedScopes = jsonResponse.scope?.length
+    ? jsonResponse.scope?.split(' ')
+    : [];
```

Comment:
Small fix to prevent values like this `['']`

Reply:
i don't get it? like if scope === ' '?

Reply:
ye, scope from server is sent as an empty string `''`